src/Security/Voter/TerritorialCouncil/AccessVoter.php

```diff
@@ -15,11 +15,20 @@ protected function doVoteOnAttribute(string $attribute, Adherent $adherent, $sub
     {
         $membership = $adherent->getTerritorialCouncilMembership();
 
-        if ($membership && $subject instanceof TerritorialCouncil) {
-            return $membership->getTerritorialCouncil() === $subject;
+        if (!$membership instanceof TerritorialCouncilMembership) {
+            return false;
         }
 
-        return $membership instanceof TerritorialCouncilMembership;
+        $territorialCouncil = $membership->getTerritorialCouncil();
+        if (!$territorialCouncil->isActive()) {
+            return false;
+        }
+
+        if ($subject instanceof TerritorialCouncil) {
```

Comment:
when do you pass $subject for this voter?

Reply:
Currently nowhere, as in case of `PoliticalCommitteeAccessVoter`.

Reply:
so we can remove it?

Reply:
Done